src/lib/swapper/utils.ts

```diff
@@ -2,26 +2,32 @@ import type { SwapErrorRight } from '@shapeshiftoss/swapper'
 import { makeSwapErrorRight, SwapErrorType, SwapperName } from '@shapeshiftoss/swapper'
 import type { Result } from '@sniptt/monads'
 import { Err, Ok } from '@sniptt/monads'
-import type { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
-import type { ISetupCache } from 'axios-cache-adapter'
-import { setupCache } from 'axios-cache-adapter'
+import Axios, { type AxiosInstance, type AxiosRequestConfig, type AxiosResponse } from 'axios'
+import { setupCache } from 'axios-cache-interceptor'
 import type { FeatureFlags } from 'state/slices/preferencesSlice/preferencesSlice'
 
 import { isCrossAccountTradeSupported } from '../../state/helpers'
 import { AsyncResultOf, isTruthy } from '../utils'
 
-const getRequestFilter = (cachedUrls: string[]) => (request: Request) =>
-  !cachedUrls.some(url => request.url.includes(url))
+const getRequestFilter = (cachedUrls: string[]) => (config: AxiosRequestConfig) =>
+  !cachedUrls.some(url => config.url?.includes(url))
 
-export const createCache = (maxAge: number, cachedUrls: string[]): ISetupCache => {
+export const createCache = (
+  maxAge: number,
+  cachedUrls: string[],
+  axiosConfig: AxiosRequestConfig,
+): AxiosInstance => {
   const filter = getRequestFilter(cachedUrls)
-  return setupCache({
-    maxAge,
-    exclude: { filter, query: false },
-    clearOnStale: true,
-    readOnError: false,
-    readHeaders: false,
+  const axiosInstance = Axios.create(axiosConfig)
+
+  setupCache(axiosInstance, {
+    ttl: maxAge,
+    cachePredicate: cacheResponse => filter(cacheResponse.config),
+    interpretHeader: false,
```

Comment:
[interpretHeader](https://axios-cache-interceptor.js.org/config/request-specifics#cache-interpretheader): `If activated, when the response is received, the ttl property will be inferred from the requests headers. As described in the MDN docs and HTML specification.` 

Setting this to false as we set our own TTL. Equivalent to the previous `readHeaders: false`.